src/globals/middleware/temporal.middleware.ts

```
@@ -5,31 +5,97 @@ export interface TemporalMiddlewareConfig {
+
+function rejectDebounceState(state: DebounceState, error: Error) {
+  if (state.timeoutId) {
+    clearTimeout(state.timeoutId);
+    state.timeoutId = undefined;
+  }
+
+  const { rejectList } = state;
+  state.resolveList = [];
+  state.rejectList = [];
+  state.latestInput = undefined;
+  rejectList.forEach((reject) => reject(error));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)